Gemfile.lock

```diff
@@ -812,4 +828,4 @@ RUBY VERSION
    ruby 3.0.3p157
 
 BUNDLED WITH
-   2.1.4
+   2.2.32
```

Comment:
Can you revert this again, please?

Reply:
> This seems to be a good choice at the moment but I'm wondering if we want to opt for Hotwire instead. https://hotwired.dev/

Hotwire is okay but StimulusReflex, I feel, is better. It is surely more powerful than hotwire because cable_ready has a alot of DOM events we can trigger from the server. it's easier to understand how to use Reflex. And I like the pattern's you get with StimulusReflex. You don't need to have the code in your controller action but can simply manipulate the DOM from a reflex action, which is just a ruby method.

Also, StimulusReflex has been here for longer than HOTWIRE. It has had more time to smooth out some of the rough edges and has already been used by alot of production apps. We already are using cable_ready, I think, pivoting now to HOTWIRE will just cause confusion in the future.